projects/control-service/projects/job-base-image/Dockerfile-data-job-base

```diff
@@ -27,6 +27,11 @@ RUN set -ex \
       && ldconfig \
       && apt-get purge -y --auto-remove curl unzip
 
+# Setup default logging with vdk-structlog
```

Comment:
I am uncertain this is the correct place. I think you want to ensure that all jobs deployed by the control service have "Cloud" settings but  this is just one base image of many. So it won't work really. You have to set it in every base image which is not feasible. 
 

I was thinking LOG_CONFIG_TYPE which can act as "aggregator" of logging configuration values 

LOG_CONFIG_TYPE="CLOUD" would set format and all config options that are suitable for cloud in a single place in vdk-struct-log

Reply:
So I should have something like below?

```python
    @hookimpl
    def vdk_initialize(self, context: CoreContext):
        config_type = context.configuration.get_value("log_config_type")
        if config_type == "CLOUD":
            # hardcode format, etc.
```

Reply:
Yeah , something like that.